documentation/puppet-api/v3/compile.markdown

```diff
@@ -36,7 +36,9 @@ The request body must look like:
   "transaction_uuid": "<uuid string>",
   "job_id": "<id string>",
   "options": { "capture_logs": <true/false>,
-               "log_level": <one of debug/info/warning/err>}
+               "log_level": <one of debug/info/warning/err>
+               "bolt": <true/false> Whether or not to attempt to load bolt
+               "boltlib": <String> Path to bolt-modules to prepend to modulepath}
```

Comment:
Why is this settable by the endpoint it feels like puppet-server configuration? It seems wrong to pass a path to a file on disk over http but I suppose when you're literally accepting arbitrary code it's at least less of a security issue.

Reply:
Fundamentally managing this is going to be kind of a pain. I explicitly asked skeletor team about where to manage it and the thought was we would manage it in orchestrator. If we can read pe-puppetserver.conf values easily here it probably makes sense to get it from there.